Source/RocketSoundEnhancement/PartModules/RSE_Module.cs

```diff
@@ -338,5 +296,64 @@ public virtual void OnDestroy()
             }
             Destroy(AudioParent);
         }
+
+        public override void OnLoad(ConfigNode node)
+        {
+            base.OnLoad(node);
+
+            if (part?.partInfo?.partPrefab != null)
+            {
+                var prefab = part.partInfo.partPrefab.FindModuleImplementing<RSE_Module>();
```

Comment:
This doesn't work if there's more than one RSE module on the part.  The old code had some disambiguating logic based on the actual module name (one of the derived types) and the moduleID field).

I think you can fix this by calculating the index of the current module and then retrieving the module at that index from the prefab.

And actually this whole thing wouldn't be necessary if we lean into unity serialization from the prefab. Though that's not always easy if it involves types that are only defined in this assembly.